lib/Agrammon/Formula/Builtins.pm6

```diff
@@ -33,6 +33,22 @@ sub get-builtins is export {
                     unless $filter-group ~~ Agrammon::Outputs::FilterGroupCollection;
             $filter-group.add(+$additor)
         },        
+        # Select only values in filter group 'a' 
+        # where values in filter group 'crit' are greater than 0
+        selectStrict => -> $filter-group-a, $filter-group-crit {
+            die "add operator expects a filter group as its first argument"
+                    unless  $filter-group-a ~~ Agrammon::Outputs::FilterGroupCollection &&
+                            $filter-group-crit ~~ Agrammon::Outputs::FilterGroupCollection;
```

Comment:
This error text needs updating (and in the other operator), otherwise it can be very confusing when we get errors. Should say "selectStrict function expects its arguments to be filter groups" or similar.

Reply:
Damn, I'm so sloppy. Thanks for spotting.

Reply:
@jnthn sorry, too early for review, need to correct this die message even further

Reply:
now, it should be ok